### PR TITLE
 Issue #ED-209 fix: Translation Issue of "My Report" text in the dashboard is fixed

### PR DESCRIPTION
**Pull request by mithun30052001**
# SunbirdEd - Portal
---

### Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
---

#### Example
- [x] Applicable
- [ ] Not applicable

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/resourcebundles/data/consumption/as.properties [hunk 1]
```diff
@@ -1930,4 +1930,5 @@ frmelmnts.lbl.confirmEvidenceDelete = Are you sure you want to delete the attach
 frmelmnts.lbl.sectionName = Section name
 frmelmnts.lbl.enterResponse = Enter your response
 frmelmnts.lbl.lastModifiedOn = Last modified on
-frmelmnts.lbl.enableDebugMode = ডিবাগ মোড সক্ষম কৰক
\ No newline at end of file
+frmelmnts.lbl.enableDebugMode = ডিবাগ মোড সক্ষম কৰক
```
Comment (princegupta1131) [inline src/app/resourcebundles/data/consumption/as.properties:1933]: @mithun30052001 please fix the whitespace

File: src/app/resourcebundles/data/consumption/mr.properties [hunk 2]
```diff
@@ -1902,4 +1902,5 @@ frmelmnts.lbl.sectionName = Section name
 frmelmnts.lbl.enterResponse = Enter your response
 frmelmnts.lbl.lastModifiedOn = Last modified on
 frmelmnts.lbl.enableDebugMode = डीबग मोड सक्षम करा
-frmelmnts.scttl.userrole1 = Add user role
\ No newline at end of file
+frmelmnts.scttl.userrole1 = Add user role
```
Comment (princegupta1131) [inline src/app/resourcebundles/data/consumption/mr.properties:1905]: @mithun30052001 please fix the whitespace

File: src/app/resourcebundles/data/consumption/ur.properties [hunk 3]
```diff
@@ -1912,4 +1912,5 @@ frmelmnts.lbl.sectionName = Section name
 frmelmnts.lbl.enterResponse = Enter your response
 frmelmnts.lbl.lastModifiedOn = Last modified on
 frmelmnts.lbl.enableDebugMode = ڈیبگ وضع کو فعال کریں
-frmelmnts.scttl.userrole1 = Add user role
\ No newline at end of file
+frmelmnts.scttl.userrole1 = Add user role
```
Comment (princegupta1131) [inline src/app/resourcebundles/data/consumption/ur.properties:1915]: @mithun30052001 please fix the white space

File: src/app/resourcebundles/data/consumption/as.properties (hunk 1 above)
Comment (mithun30052001) [reply src/app/resourcebundles/data/consumption/as.properties:1933]: Sure prince

**Pull request closed without merging**